### PR TITLE
Start fixing warnings of undefined array key access

### PR DESCRIPTION
**Pull request by dmolineus**
This pull request is the initial start to fix warnings of undefined array key access when using isotope in PHP 8.

Disclaimer: This is not a complete fix just a result of trying to use PHP 8.1 and isotope 2.8.x-dev with Contao 4.13. It may be a starting point to generally fix the issues in Isotope.

### CODE REVIEW

File: system/modules/isotope/config/config.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 /**
  * Backend modules
  */
-if (!\is_array($GLOBALS['BE_MOD']['isotope']))
+if (!isset($GLOBALS['BE_MOD']['isotope']) || !\is_array($GLOBALS['BE_MOD']['isotope']))
```
Comment (fritzmg) [inline system/modules/isotope/config/config.php]:
```suggestion
if (!\is_array($GLOBALS['BE_MOD']['isotope'] ?? null))
```

File: system/modules/isotope/drivers/DC_ProductData.php [hunk 2]
```diff
@@ -80,7 +80,7 @@ public function __construct($strTable)
         /** @var SymfonySession $objSession */
         $objSession = System::getContainer()->get('session');
         $arrClipboard = $objSession->get('CLIPBOARD');
-        if ($arrClipboard[$strTable]['mode'] === 'cutAll' && \Input::get('act') !== 'cutAll') {
+        if (isset($arrClipboard[$strTable]['mode']) && $arrClipboard[$strTable]['mode'] === 'cutAll' && \Input::get('act') !== 'cutAll') {
```
Comment (fritzmg) [inline system/modules/isotope/drivers/DC_ProductData.php]:
```suggestion
        if ($arrClipboard[$strTable]['mode'] ?? null === 'cutAll' && \Input::get('act') !== 'cutAll') {
```

**Comment by aschempp** [comment]
Can someone explain to me why the `deleteConfirm` should ever be empty? They are always defined in the fallback language?

**Comment by fritzmg** [comment]
@aschempp you removed the changes for `$GLOBALS['TL_LANG']['tl_iso_attribute']['deleteConfirm']` etc., which will cause the problem again.

**Pull request merged**